includes/payment-methods/class-upe-payment-gateway.php

```diff
@@ -991,6 +991,8 @@ public function get_upe_available_payment_methods() {
 		$available_methods[] = Sepa_Payment_Method::PAYMENT_METHOD_STRIPE_ID;
 		$available_methods[] = P24_Payment_Method::PAYMENT_METHOD_STRIPE_ID;
 		$available_methods[] = Link_Payment_Method::PAYMENT_METHOD_STRIPE_ID;
+		$available_methods[] = Affirm_Payment_Method::PAYMENT_METHOD_STRIPE_ID;
```

Comment:
Getting 
> ERROR Amount must be no less than $50.00 USD for the Affirm payment method. (amount_too_small)

response from Stripe when creating intent on checkout load. The UPE element brakes because of that 
![image](https://github.com/Automattic/woocommerce-payments/assets/24650518/680adcfb-6517-4cdb-9fb4-40b45226266a)


@mgascam @kalessil please advise how to best add available method based on price